docs/en/samples/fuse_affinity.md

```diff
@@ -0,0 +1,222 @@
+
+**Check the Deployment to be created**
+
+Below is the YAML snippet for a simple Deployment with two copies and the selector tag `app=nginx-test`. Deployments are configured with `PodAntiAffinity` to ensure that the scheduler does not schedule all replicas on the same node.
+
+```
+$ cat<<EOF >nginx.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+spec:
+  selector:
+    matchLabels:
+      app: nginx-test
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: nginx-test
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - nginx-test
+            topologyKey: "kubernetes.io/hostname"
+      containers:
+        - name: nginx
+          image: nginx
+          volumeMounts:
+            - mountPath: /data
+              name: hbase-vol
+      volumes:
+        - name: hbase-vol
+          persistentVolumeClaim:
+            claimName: hbase
+EOF
+```
+
+**Create the application**
+
+```
+$ kubectl create -f nginx.yaml
+deployment.apps/nginx created
+```
+
+**Check the status of pods**
+
+```
+$ kubectl get po -owide
+NAME                    READY   STATUS    RESTARTS   AGE   IP             NODE                      NOMINATED NODE   READINESS GATES
+hbase-fuse-4jfgl        1/1     Running   0          41m   172.16.0.101   cn-beijing.172.16.0.101   <none>           <none>
+hbase-master-0          2/2     Running   0          64m   172.16.0.101   cn-beijing.172.16.0.101   <none>           <none>
+hbase-worker-0          2/2     Running   0          64m   172.16.0.101   cn-beijing.172.16.0.101   <none>           <none>
+nginx-766564fc7-8vz4s   0/1     Pending   0          41m   <none>         <none>                    <none>           <none>
+nginx-766564fc7-rtmwh   1/1     Running   0          41m   10.73.0.135    cn-beijing.172.16.0.101   <none>           <none>
+```
+
+It can be seen that the Fuse client is only started on the node `cn-beijing.172.16.0.101` with the `select-node=true` label.
+
+One of the two pods (`nginx-766564fc7-rtmwh`) is started on node `cn-beijing.172.16.0.101` because of `PodAntiAffinity` between pods, the other pod (`nginx-766564fc7-8vz4s`) which should be started on another node `cn-beijing.172.16.0.99`, but due to the limitation of `spec.fuse.nodeSelector`, it cannot be scheduled and is in pending state.
+
+**start all pods**
+
+In order to prove that the pod is not scheduled because of the limitation of `spec.fuse.nodeSelector`, we also mark the node `cn-beijing.172.16.0.99` with `select-node=true`, you can find the Fuse Pod and Nginx Pod of this node All started.
```

Comment:
```suggestion
In order to prove that the pod is not scheduled because of the limitation of `spec.fuse.nodeSelector`, we also label the node `cn-beijing.172.16.0.99` with `select-node=true`, you can find the Fuse Pod and Nginx Pod of this node All started.
```